test/format/backup.c

```diff
@@ -590,17 +590,17 @@ backup(void *arg)
 
             } else
                 testutil_copy_file(session, key);
-            trace_msg(session, "Backup #%u copy file %s stop", counter, key);
+            trace_msg(session, "Backup #%u copy file %s stop, ret=%d", counter, key, ret);
```

Comment:
Don't we know for sure that ret is 0 here?

Reply:
that wouldn't be the case in the while loop iteration

Reply:
yes that's true. It wouldn't be useful in that instance.